src/.nuget/Microsoft.NETCore.ILAsm/Microsoft.NETCore.ILAsm.builds

```diff
@@ -1,89 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), packaging.props))\packaging.props" />
```

Comment:
If you want to have different modes for packages I think it would be clearer to describe those modes with properties rather than importing different targets with names that don't correspond to how they are different.